frontend/assets/javascripts/src/modules/form/address/subdivision.js

```diff
@@ -0,0 +1,50 @@
+define(['$'], function ($) {
+    'use strict';
+
+    var HIDE_CONTENT_VISUALLY_CLASSNAME = 'u-h';
+    var UNITED_STATES_STRING = 'united states';
```

Comment:
Would be good to document what the country specific changes do / why they are needed.